cloud-operator/templates/rolebinding.yaml

```
@@ -26,4 +26,18 @@ subjects:
 roleRef:
   kind: Role
   name: hubble-client-certs-reader
-  apiGroup: rbac.authorization.k8s.io
\ No newline at end of file
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: illumio-cloud-operator-secret-reader-binding
```

Use the same name as the RoleBinding we create in `kube-system` above, for consistency.
```suggestion
  name: cloud-operator-read-hubble-client-certs
```